codegen/masm/src/codegen.rs

```diff
@@ -458,6 +482,12 @@ impl<'ast> Backend<'ast> {
         self.writer.drop();
         self.writer.drop();
     }
+
+    /// Emits code to load `g`, the trace domain generator.
```

Comment:
should we also add a suffix to g here?

Reply:
I can change from `g` to something else, I'm not sure what would be a good name though. Any suggestions?